source/rpt/rpt.cc

```diff
@@ -65,11 +61,13 @@ RPT<dim>::calculate()
             detectors[i_detector],
             rpt_parameters);
 
-          // Calculate count and print it in terminal
+          // Calculate count and print it in terminal or store is if tuning
+          // is enable
           double count = particle_detector_interactions.calculate_count();
-          std::cout << "Count for particle position " << i_particle
-                    << " and detector " << i_detector << " : " << count
-                    << std::endl;
+          if (rpt_parameters.initial_param.tuning == true)
+            calculated_counts.push_back(count);
+          else
+            std::cout << count << std::endl;
```

Comment:
Into a file? It's stored in csv if export is enable. Here it stored into a vector.
This part is weird because before the tuning, there was no storing of counts in vector.
In the rpt_positions branch I changed it and counts will always be stored in calculated_count vector.

Reply:
OK i understand. Good